TROUBLESHOOTING.md

```diff
@@ -99,3 +100,14 @@ If you're getting the following error while using this library:
 It is most likely due to a linebreak in your API key. Passing your key through `trim` should fix this:
 
 `$apiKey = trim($apiKey)`
+
+<a name="request-body"></a>
+## Viewing the Request Body
+
+When debugging or testing, it may be useful to exampine the raw request body to compare against the [documented format](https://sendgrid.com/docs/API_Reference/api_v3.html).
```

Comment:
Spelling mistake, `exampine` should be `examine` 👍

Reply:
Done